handlers/response/writer_test.go

```diff
@@ -0,0 +1,75 @@
+package response
```

Comment:
Consider putting tests in package 'response_test' to make the tests external to the package, while still in the same directory.